Tests/BugsnagTests/PayloadTransformerTests.swift

```diff
@@ -84,7 +93,11 @@ class PayloadTransformerTests: XCTestCase {
     }
 
     func testThatSeverityIsCorrect() {
-        XCTAssertEqual(payload["events"]?[0]?["severity"]?.string, "error")
+        XCTAssertEqual(payload["events"]?[0]?["severity"]?.string, self.severity.rawValue)
```

Comment:
Related to my comment above, to be on the safe side, consider comparing against `.warning` instead of the variable here.